xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/layouts/table/LayoutTableHeaderFilters.vue

```diff
@@ -91,6 +92,14 @@ export default {
   */
   width: 100vw;
 }
+.layout-table .column-filters th:not(:empty) {
+  border: 1px solid #ddd;
```

Comment:
Square corners on filter inputs is not consistent with XWiki style: we use rounded corners on form fields everywhere else.